GA/utils.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+class Utils:
+    # Dictionary mapping slot numbers to time intervals in 12-hour format
```

Comment:
Should be more descriptive. Class will only have one responsibility, and only one reason to change.

Mention all the features your class gives us.

Reply:
We will be including the lab times in this slot or just add two slot? The current time_slots dictionary might have overlapping intervals in some cases when dealing with labs. This can cause issues if intervals are added or modified without caution.